pkgs/top-level/all-packages.nix

```diff
@@ -17553,12 +17553,9 @@ with pkgs;
   tinycc = darwin.apple_sdk_11_0.callPackage ../development/compilers/tinycc { };
 
   tinygo = callPackage ../development/compilers/tinygo {
-    llvmPackages = llvmPackages_14;
+    llvmPackages = llvmPackages_15;
```

Comment:
why not 16, it's the one upstream defaults to, right?

Reply:
well spotted,  release notes say that 16 is default now, but it doesn't build with 16. I only did 0.30 update yesterday so haven't had a chance to look into it.

Reply:
if 15 works, no reason to delay this PR to make 16 work. That could be left for the future

Reply:
fixed build with 16